### ISSUE TITLE
Can not run ember serve after install - Cannot read property 'addon' of undefined

### ISSUE THREAD

**Issue by samuil4** [ember-cli/ember-cli#5467]
OS: Win 10
Node version: 5.6.0

After installed ember-cli the ember server wont start with the error Cannot read property 'addon' of undefined:
```
D:\www>ember new new-ember-test
version: 1.13.15
installing app
  create .bowerrc
  create .editorconfig
  create .ember-cli
  create .jshintrc
  create .travis.yml
  create .watchmanconfig
  create README.md
  create app\app.js
  create app\components\.gitkeep
  create app\controllers\.gitkeep
  create app\helpers\.gitkeep
  create app\index.html
  create app\models\.gitkeep
  create app\router.js
  create app\routes\.gitkeep
  create app\styles\app.css
  create app\templates\application.hbs
  create app\templates\components\.gitkeep
  create bower.json
  create config\environment.js
  create ember-cli-build.js
  create .gitignore
  create package.json
  create public\crossdomain.xml
  create public\robots.txt
  create testem.json
  create tests\.jshintrc
  create tests\helpers\destroy-app.js
  create tests\helpers\module-for-acceptance.js
  create tests\helpers\resolver.js
  create tests\helpers\start-app.js
  create tests\index.html
  create tests\integration\.gitkeep
  create tests\test-helper.js
  create tests\unit\.gitkeep
  create vendor\.gitkeep
Successfully initialized git.
Installed packages for tooling via npm.
Installed browser packages via Bower.

D:\www>cd new-ember-test

D:\www\new-ember-test>ember serve
version: 1.13.15
Cannot read property 'addon' of undefined
TypeError: Cannot read property 'addon' of undefined
    at Class.module.exports.lintTree (D:\www\new-ember-test\node_modules\ember-cli-qunit\index.js:192:43)
    at D:\www\new-ember-test\node_modules\ember-cli\lib\broccoli\ember-app.js:480:20
    at Array.map (native)
    at EmberApp.addonLintTree (D:\www\new-ember-test\node_modules\ember-cli\lib\broccoli\ember-app.js:478:36)
    at EmberApp.appTests (D:\www\new-ember-test\node_modules\ember-cli\lib\broccoli\ember-app.js:967:30)
    at EmberApp.test (D:\www\new-ember-test\node_modules\ember-cli\lib\broccoli\ember-app.js:953:26)
    at EmberApp.toArray (D:\www\new-ember-test\node_modules\ember-cli\lib\broccoli\ember-app.js:1396:61)
    at EmberApp.toTree (D:\www\new-ember-test\node_modules\ember-cli\lib\broccoli\ember-app.js:1411:30)
    at module.exports (D:\www\new-ember-test\ember-cli-build.js:23:14)
    at Class.module.exports.Task.extend.setupBroccoliBuilder (D:\www\new-ember-test\node_modules\ember-cli\lib\models\builder.js:57:19)
```

**Comment by nickschot** [comment]
Having exactly the same issue here (since somewhere today). Ember-cli 1.13.14 in this case.

**Comment by locks** [comment]
From @rwjblue:

> rwjblue [8:08 PM] 
@emma force ember-cli-qunit@1.2.0 and please report an issue, I'm away from my computer ATM but will fix it asap
[8:08] 
This is almost certainly related to a recent update to ember-cli-QUnit I did an hour or so ago

(The issue is this one, so it's been reported ;)

**Comment by rwjblue** [comment]
Should be fixed with ember-cli-qunit@1.2.3

**Comment by rwjblue** [comment]
Ok, finally good for node 0.10/0.12 in ember-cli-qunit@1.2.4.  Sorry for the issues y'all!
